### ISSUE TITLE
LearnDash Course Pages go blank when Landing Pages are enabled

### ISSUE THREAD

**Issue by markwmim** [elementor/elementor#13728]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Description

I saw there was another post on this but I wanted to provide more details. I have a single page template that I was using for my courses page from LearnDash. The template just has the Post Content Widget with nothing else on the page. When Landing Pages is enabled, nothing shows on the courses page. When it is disabled everything comes back. I'm using Kadence Theme but I have also tried this on the default Wordpress theme and the issue still occurs. I've tried this on two of my site that are running LearnDash and it occurs on both them using the same steps.  One LearnDash Site uses the default name Courses the other site using a custom name called Guides. 

I also tried this by just using the standard wordpress/theme page template and when Landing Page is enabled the courses post content does not appear. If I manually enter a url for a lesson (which is a post and not a page, the post content appears as it should and I can navigate and continue to the next/previous lesson. Once you go back to the Course Page, the Page content does not show. 

## Steps to reproduce
Enable Landing Page
View a LearnDash Course Page

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by DorShahar** [comment]
Hi
Thanks for reporting.
We are aware of the problem and working on a fix.
I'm merging this issue with #13706  for better tracking.

Thanks.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.1.1

Feel free to update

Thanks!
